src/mlpack/methods/ne/tasks.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file tasks.hpp
+ * @author Bang Liu
+ *
+ * Definition of Population class.
+ */
+#ifndef MLPACK_METHODS_NE_POPULATION_HPP
+#define MLPACK_METHODS_NE_POPULATION_HPP
+
+#include <cstddef>
+
+#include <mlpack/core.hpp>
+#include <mlpack/methods/ann/performance_functions/cee_function.hpp>
+
+#include "link_gene.hpp"
+#include "neuron_gene.hpp"
+#include "genome.hpp"
+
+namespace mlpack {
+namespace ne {
+
+/**
+ * This class defines task xor.
+ */
+template<typename FitnissFunction = MeanSquaredErrorFunction>
+class TaskXor {
+ public:
```

Comment:
I really like the idea of the task class, I think that makes really easy to write problems that can be solved by the ne methods.